docs/units/format.rst

```diff
@@ -52,8 +52,10 @@ attributes within format strings::
     '10.500 in km'
 
 This might not work well with LaTeX strings, in which case it would be better
-to use the `Quantity.to_string() <astropy.units.Quantity.to_string()>`
-method::
+to use the `Quantity.to_string() <astropy.units.Quantity.to_string()>` method.
+This method can be enhanced with the formatter parameter, providing added
+format flexibility. It allows you to specify a dict, callable, or format_spec
+string, further refining the LaTeX format representation::
```

Comment:
We should link to the `numpy.array2string()` and `Table` column formatting documentation as much as possible and only highlight the differences here.